api/src/main/java/graphql/nadel/Nadel.java

```diff
@@ -77,7 +75,6 @@
     final ServiceExecutionHooks serviceExecutionHooks;
     final PreparsedDocumentProvider preparsedDocumentProvider;
     final ExecutionIdProvider executionIdProvider;
-    final IntrospectionRunner introspectionRunner;
```

Comment:
New engine gets supplied one directly to `NextgenEngine` constructor. At some point we'll create a tighter coupling between the two classes/modules again so that avoid the need to directly construct `NextgenEngine`.